### PR TITLE
Create GCD_algo.java

### PR DESCRIPTION
**Pull request by sktagrwl**

### CODE REVIEW

File: Java/GCD_algo.java [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+			gcd = b;
+		}
+		else if (b>a)
+		{
+			gcd = a;
+		}
+		
+		while(true)
+		{
+			if(a% gcd ==0 && b% gcd==0)
```
Comment (chankruze) [inline Java/GCD_algo.java:25]: Please respect whitespaces!

File: Java/GCD_algo.java [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+		// TODO Auto-generated method stub
+		
+		Scanner sc = new Scanner(System.in);
+		int a = sc.nextInt();
+		int b = sc.nextInt();
+		int gcd = 0;
+		
+		
+		if (a>b)
+		{
```
Comment (chankruze) [inline Java/GCD_algo.java:15]: Consider reading this before you commit to VCS: https://google.github.io/styleguide/javaguide.html

**Review by chankruze — COMMENTED** [review]
No words!

**Comment by k40t1k-sys** [comment]
This is a very basic code in regard of contribution hence we cannot accept. You can refer to the problem list present in the main repository for some ideas or create something of your own.

**Pull request closed without merging**